xds/internal/server/listener_wrapper.go

```diff
@@ -185,6 +208,16 @@ type listenerWrapper struct {
 	mode ServingMode
 	// Filter chains received as part of the last good update.
 	filterChains *xdsclient.FilterChainManager
+	// rdsHandler is used for any dynamic RDS resources specified in a LDS
+	// update.
+	rdsHandler *rdsHandler
+	// rdsUpdates are the RDS resources received from the management
+	// server, keyed on the RouteName of the RDS resource.
+	rdsUpdates map[string]xdsclient.RouteConfigUpdate // TODO: if this will be read in accept, this will need a read lock as well.
+	// updateCh is a channel for XDSClient LDS updates.
+	updateCh chan ldsUpdateWithError
```

Comment:
Nit: now that we have an `rdsUpdateCh`, it would be nice to name this `ldsUpdateCh`. Thanks.